plugins/node/opentelemetry-instrumentation-pg/src/instrumentation.ts

```diff
@@ -88,9 +91,14 @@ export class PgInstrumentation extends InstrumentationBase<PgInstrumentationConf
     idle: 0,
     pending: 0,
   };
+  private _semconvStability: SemconvStability = SemconvStability.OLD;
```

Comment:
nit: I think you don't actually need the initial value here, per the example usage documented here: https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-instrumentation/src/semconvStability.ts#L53

Obviously, is fine to have it.

Reply:
Ah okay, I used the http one as a guide:
https://github.com/open-telemetry/opentelemetry-js/blob/f2cfd1327a5b131ea795301b10877291aac4e6f5/experimental/packages/opentelemetry-instrumentation-http/src/http.ts#L98

Reply:
Fair.